Data/NF/Internal.hs

```diff
@@ -17,11 +21,17 @@
 -- Stability   :  experimental
 -- Portability :  portable
 --
--- This module provides the constructor for the type 'NF', allowing
--- you to create values of type 'NF' without any runtime overhead
--- if you fulfill a proof obligation that the value is already in
--- normal form.
-module Data.NF.Internal(NF(..)) where
+-- This module is intended for internal use.
```

Comment:
Uh, the key difference between this and unsafe is internal blasts the coercion into public view.  So might as well call a spade a spade.